src/main/java/it/gov/pagopa/pu/workflow/wf/paymentsreporting/pagopa/config/OrganizationPaymentsReportingPagoPaFetchWfConfig.java

```diff
@@ -1,4 +1,4 @@
-package it.gov.pagopa.pu.workflow.wf.pagopa.paymentreporting.config;
+package it.gov.pagopa.pu.workflow.wf.paymentsreporting.pagopa.config;
```

Comment:
don't invert paymentsReporting and pagopa package.
it was do in order to collect under pagopa all pagopa related fetches